propagation/onboarding.md

```diff
@@ -0,0 +1,76 @@
+# Algorithm Onboarding Service
+
+APEx offers an algorithm onboarding service as part of its propagation services to ensure that project results remain
+accessible to the EO community beyond the project's completion. This service aims to sustain and promote the use of
+algorithms developed during ESA projects, ensuring their longevity and continued utilization.
+
+For engagement with stakeholders of the algorithm, APEx collaborates closely with
+the [Stakeholder Engagement Facility](https://esa-sef.eu/). This initiative will provide outreach and training towards
+users, to maximize the uptake of the onboarded algorithms.
+
+By leveraging APEx’s Algorithm Onboarding service, you can ensure that your project’s innovations continue to benefit
+the
+EO community long after your project ends.
+
+## Service Overview
+
+### Hosting Platform Onboarding Support
+
+APEx supports your project in selecting the appropriate hosting environment for your algorithm based on
+the [APEx Interoperability and Compliance Guidelines](../interoperability/algohostingenv.md). APEx also assists in the
+onboarding process, ensuring that your algorithm is available for execution on the selected platform and ready for
+further integration into the APEx Algorithm Catalogue.
+
+:::{.callout-tip}
+A full list of the known APEx-compliant algorithm hosting platforms is available [here](index.qmd#what-are-the-platforms-to-choose-from).
+:::
+
+### Automated Testing and Benchmarking
+
+To maintain the integrity of the algorithms, APEx conducts automated testing and benchmarking in collaboration with the
+respective project teams. This ensures that the algorithms remain functional and accessible over time. APEx offers this testing service for free, which reduces the maintenance burden on projects.
+
+To run the tests, APEx needs permission from the hosting platform, and an account with sufficient credits. Collaboration of the platform may be required to fulfill this requirement.
+
+### APEx Algorithm Catalogue
+
+The Algorithm Onboarding service supports result propagation through an algorithm catalogue, facilitating easy
+browsing and access to available algorithms. Users and other projects can also easily onboard their own algorithms,
+fostering collaboration and innovation within the EO community.
+
+The algorithm catalogue allows visitors to execute algorithms on the corresponding APEx-compliant hosting environment.
+For more details, please refer to [this section](#execute-a-service-from-the-apex-algorithm-catalogue).
+
+## Condiserations and Project Responsiblities
+
+### Algorithm Maintenance
+
+While APEx ensures the long-term availability of the algorithm within the catalogue, the creation, maintenance, and
+updating of the algorithm remain the responsibility of the original developer or the open-source community.
+
+### Algorithm Execution
+
+Algorithm execution is carried out one of the respective APEx-compliant algorithm hosting platform. By default, APEx
```

Comment:
in the near future there will be a algo execution GUI that will allow runining algos in a platform as long as the user is authenticated and/or uses a free tier or credit subscription.. we shoulda dd this

Reply:
Similar to maintenance, we want to make it clear that APEx itself doesn't execute anything, and is thus not responsible for errors and bugs.
So if we mention the GUI, we would also need to add that it is _only_ a user interface, and that actual execution is happening elsewhere.

Reply:
Added suggestion to the text.